src/com/esotericsoftware/kryo/Kryo.java

```diff
@@ -225,6 +227,7 @@ public Kryo (ClassResolver classResolver, ReferenceResolver referenceResolver) {
 		addDefaultSerializer(Arrays.asList().getClass(), ArraysAsListSerializer.class);
 		addDefaultSerializer(void.class, new VoidSerializer());
 		addDefaultSerializer(PriorityQueue.class, new PriorityQueueSerializer());
+		addDefaultSerializer(ConcurrentHashMap.KeySetView.class, new KeySetViewSerializer());

```

Comment:
Please remove the serializer registration. It breaks backwards compatibility. If we add the serializer to Kryo 5, users will have to register it manually. It can only be registered as a default serializer in the next major version.